config/default.yml

```diff
@@ -761,6 +761,16 @@ RSpec/VariableName:
   VersionChanged: '1.43'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/VariableName
 
+RSpec/VerifiedDoubleReference:
+  Description: Checks for consistent verified double reference style.
+  Enabled: false
```

Comment:
I'd suggest [`pending`, this special status](https://docs.rubocop.org/rubocop/versioning.html) was introduced specifically for this purpose.